### PR TITLE
Fix CI build warnings

### PR DESCRIPTION
**Pull request by hannobraun**

### CODE REVIEW

File: src/i2c.rs [hunk 1]
```diff
@@ -277,6 +275,12 @@ fn calculate_timing(
     an_filter: bool,
     dnf: u8,
 ) -> I2cTiming {
+    // We get an "unused" warning here when running `cargo test`. I haven't
+    // looked into it, but I suspect that is only used when `std` isn't
+    // available, which would explain that behavior.
```
Comment (rfuest) [inline src/i2c.rs:280]: This problem is mentioned in the `micromath` docs: https://docs.rs/micromath/1.1.0/micromath/#unused-import-warnings-when-linking-std

Comment (hannobraun) [reply src/i2c.rs]: Thanks, @rfuest! I updated the comment to address that.

**Comment by hannobraun** [comment]
Rebased on latest master and pushed a new commit that updates a comment.

**Review by mvertescher — APPROVED** [review]
Looks good!

**Pull request merged**